crates/fuel-streams-core/src/utxos/mod.rs

```diff
@@ -0,0 +1,16 @@
+pub mod subjects;
+pub mod types;
+
+use serde::{Deserialize, Serialize};
+pub use subjects::*;
+
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Utxo(pub Option<Vec<u8>>);
```

Comment:
This structure seems weird especially since we have some control over creating this data. I don't see why it should be optional

```suggestion
pub struct Utxo(pub Vec<u8>);
```

Reply:
yep, but pls see my question/comment above !

Reply:
added utxoId